documents/operating.md

```diff
@@ -13,10 +13,44 @@ Contents:
 ---------------------------------------------------------------------
 SECTION 1: 📚📚📚 Official Operating Agreement 📚📚📚
 ---------------------------------------------------------------------
-- [ ] Company term
-- [ ] Continuance of company
-- [ ] Rickroll
-- [ ] Game of Mario Party 3 before legal arbitration between members
+- [x] Company term
+- [x] Continuance of company
+- [x] Rickroll
+- [x] Game of Mario Party 3 before legal arbitration between members
+
+### Company Term
+BIFFUD will exist until either (1) a vote to terminate the organization
+is passed by a consensus of all Corporate Overlords or (2) the board fails to
+hold a single meeting with quorum for a period of (a consecutive year?),
+(3) everyone dies / there are no more Corporate Overlords, or (4) if all
+Corporate Overlords are straight white cis men.
+
+### Continuance of Company
+If a Corporate Overlord leaves BIFFUD, dies, or uploads their brain to the
+cloud, the ownership of the remaining corporate overlords will then be evenly
+distributed (e.g. if four overlords remain, each will own 25% of BIFFUD)
+
+### Lawsuits between BIFFUD associates
+All BIFFUD associates involved in a lawsuit must play a group game of
+Mario Party 3 before any arbitration happens.  BIFFUD will loan any
+necessary hardware needed to make this possible.
+
+### Additional notes
+It is worth explicitly stating that the Corporate Overlords will work
+tirelessly to avoid any of the following:
+
+- Giving up on our stakeholders
+- Letting anybody down who we have made a commitment to
+- Disappearing / "running around" for long periods of time without communicating
+- Deserting one another as members without formally abdicating our stake.
+
+Furthermore, they will try not to:
+
+- Make anybody cry
+- Close the company and say "goodbye", so long as there someone is out there
+willing to carry the torch forward.
+
```

Comment:
Add punctuation to the end of each bullet point. You do elsewhere. Unless this is intentional cause of the Roll?